mybackend/controller/users.js

```diff
@@ -0,0 +1,150 @@
+const User = require('../models/User');
+const { ethers } = require('ethers');
+
+class UserController {
+
+    constructor() {
+        this.usdcContractAddress = '0x1d17CBcF0D6D143135aE902365D2E5e2A16538D4';
```

Comment:
would be nice if we make this dynamic and load it from a `config` file

Reply:
Yes, that will add a layer of security. You're right.